mono_repo/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-* Add `recursive` global flag. When set, we will walk all sub-directories looking for `pubspec.yaml` files.
-
 ## 0.2.0
```

Comment:
Looks like I forgot the CHANGELOG in #38 ...

Reply:
Want to throw me some text to add in?

Reply:
* Add `before_script` support to the `travis` command. When that value is set in a `travis.yml` file, we will call the script before running any of the tasks for that package.